src/Aspire.Dashboard/Components/Controls/Chart/ChartContainer.razor.cs

```diff
@@ -79,7 +91,7 @@ private async Task UpdateDataAsync()
         while (await timer!.WaitForNextTickAsync())
         {
             _instrument = GetInstrument();
-            if (_instrument == null)
+            if (_instrument == null || PauseManager.MetricsPaused)
```

Comment:
This stops the graph from updating while you are on the page, but if you navigate away and back again then the time updates.

I think the pause manager needs to track when metrics was paused. Then when someone navigates back to the page then that time is used as the "current time" when rendering the graph.

The time scale on this page is "The last 15 minutes" (for example). When paused that time should refer to the last 15 minutes since the pause time.

Reply:
I have updated the `PauseManager` and `ChartBase` to use the paused time as `inProgressDataTime` and `_currentDataStartTime` on initialization